protocol/x/clob/memclob/memclob_grpc_streaming.go

```diff
@@ -86,3 +88,41 @@ func (m *MemClobPriceTimePriority) GetOffchainUpdatesForOrder(
 
 	return offchainUpdates
 }
+
+// GetOrderbookUpdatesForOrderRemoval returns a place order offchain message and
+func (m *MemClobPriceTimePriority) GetOrderbookUpdatesForOrderRemoval(
```

Comment:
In `MemclobPriceTimepriority` object we store a bool that indicates if we should ever ben emitting offchain events.
```
	// ---- Fields for determining if off-chain update messages should be generated ----
	generateOffchainUpdates bool
```
Can we do the same for orderbook updates? It would be nice to have all these no-op if this isn't a full node with streaming turned on.